tests/tests/regional_restart

```diff
@@ -36,7 +36,7 @@ export JNPES=6
 export WARM_START=.T.
 export NGGPS_IC=.F.
 export EXTERNAL_IC=.F.
-# DH* The correct setting would be .F.? However the official 
+# DH* The correct setting would be .F.? However the official
 #     regression test baseline uses MAKE_NH=.T.
 #export MAKE_NH=.F.
 export MAKE_NH=.T.
```

Comment:
So you don't change the MAKE_NH to be .F.? Maybe we need to change it in next PR when baseline will be updated.

Reply:
I tried to set it to F then I created a new baseline and tried to compare against it using the tests as-is (changing only this value). It didn't reproduce and I didn't understand how the test was being set up so I left it.

Reply:
The regional control is using FMS, so the history files contain fields at previous output time which do not exist in the restart run history files. Do you compare the restart files from those two runs?

Reply:
I think all I did was to change the make_nh, create a new baseline and then ran the regional_control and regional_restart tests against that baseline.